driver/src/main/java/org/neo4j/driver/Config.java

```diff
@@ -149,6 +163,16 @@ public ConfigBuilder withMinIdleTimeBeforeConnectionTest( long milliSecond )
         }
 
         /**
+         * Enable TLS in all connections with server
+         * @param value
+         */
+        public ConfigBuilder withTLSEnabled( boolean value )
```

Comment:
I don't think first-trust should be default. It seems a special snowflake. Is there prior art?

Also I think turning off TLS and relying on firewalls or AWS security groups is a valid use case.